src/widgets/__tests__/__snapshots__/EstimatedCasesChart.test.tsx.snap

```diff
@@ -17,13 +17,13 @@ exports[`<EstimatedCasesChart> dataset0 renders correctly 1`] = `
         2021-12-29
       </b>
       <div
-        className="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
+        className="MuiFormGroup-root css-1mac7pt-MuiFormGroup-root"
```

Comment:
I see that all these IDs have changed but I don't see what they are regarding... what is `css-1mac7pt-MuiFormGroup-root` labeling?